modules/cloudant/src/main/java/com/ibm/cloud/cloudant/features/ChangesOptionsHelper.java

```diff
@@ -112,7 +107,7 @@ static void validateOptions(PostChangesOptions options) {
             invalidOptions.add(String.format("filter=%s", options.filter()));
         }
         if (invalidOptions.size() > 0) {
-            throwInvalidOptionsMessageWith(String.format(" when using %s.", ChangesFollower.class.getSimpleName()), invalidOptions);
+            throwInvalidOptionsMessageWith(invalidOptions);
```

Comment:
No against it, but what's a rational behind this change?

Reply:
It was obsolete, I think the message was originally formatted here, but later it was adjusted and the `suffix` arg was never used, but it wasn't removed. This just cleans up that oversight.